requirements.txt

```diff
@@ -1,43 +1,72 @@
-alembic==1.1.0
+apipkg==1.5
+Appium-Python-Client==0.28
```

Comment:
This entire requirements file looks wrong. Where are we using appium or results or arrow in this project?